_health-care/_js/_components/name-and-general-info-section.jsx

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+
+import DateInput from './date-input';
+import FullName from './full-name';
+import MothersMaidenName from './mothers-maiden-name';
+
+class NameAndGeneralInfoSection extends React.Component {
+  render() {
+    return (
+      <div>
+        <div className="row">
+          <div className="small-12 columns">
+            <h3>Name and General Information</h3>
+          </div>
+        </div>
+        <fieldset className="usa-fieldset-inputs usa-sans">
+          <div className="row">
+            <div className="small-12 columns">
+              <h4>Veteran's Name</h4>
+              <FullName name={this.props.data.fullName}
+                onUserInput={(update) => {this.props.onStateChange('fullName', update);}} />
+              <MothersMaidenName name={this.props.data.mothersMaidenName}
+                onUserInput={(update) => {this.props.onStateChange('mothersMaidenName', update);}} />
+            </div>
+            <div className="small-12 columns">
+              <h4>Date of Birth</h4>
+              <span className="usa-form-hint usa-datefield-hint" id="dobHint">For example: 04 28 1986</span>
+              <DateInput date={this.props.data.dateOfBirth}
+                onUserInput={(update) => {this.props.onStateChange('dateOfBirth', update);}}/>
+            </div>
+            <div className="usa-input-grid usa-input-grid-medium usa-input-error">
+              <label className="usa-input-error-label" htmlFor="veteran_ssn">Social Security Number
+                <span className="usa-additional_text">Required</span>
+              </label>
+              <span className="usa-input-error-message" id="input-error-message" role="alert">Please put you number in this format xxx-xx-xxxx</span>
+              <input type="text" name="veteran[ssn]" aria-describedby="input-error-message"/>
+            </div>
+          </div>
+
+          <div className="usa-input-grid usa-input-grid-large">
+            <label htmlFor="veteran_gender">Gender</label>
+            <select name="veteran[gender]" ><option value="M">Male</option>
+              <option value="F">Female</option>
+              <option value="2">NeedOtherOfficialOptions</option>
+            </select>
+
+            <h4>Place of Birth</h4>
+            <div>
+              <div className="usa-input-grid usa-input-grid-medium">
+                <label htmlFor="veteran_city_of_birth">City</label>
```

Comment:
Yes. I know these are all broken. I actually think as we go to create a <ValidatedInput> component that will handle all this for us.